src/pages/DrowsinessSearch.jsx

```
@@ -1,12 +1,153 @@
+            className="w-full"
+          />
+        </div>
+
+        <div className="flex w-full flex-col justify-end md:basis-1/2">
+          <div className="flex w-full justify-end gap-2 md:flex-nowrap">
+            <div className="flex w-full flex-col">
+              <label className="caption-bold">검색 시작일</label>
+              <DatePicker
+                selected={startDate}
+                locale={ko}
+                dateFormat="yyyy년 MM월 dd일"
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)